### PR TITLE
Support to send extra authorization params to the OP

### PR DESCRIPTION
**Pull request by francisco-idpartner**
Update the client to support sending extra authorization params to the OP

### CODE REVIEW

File: lib/idpartner.js [hunk 1]
```diff
@@ -170,16 +171,16 @@ class IDPartner {
...
       response_type: 'code',
       client_id: clientId,
       nbf: Math.floor(new Date().getTime() / 1000),
       'x-fapi-interaction-id': uuidv4(),
       identity_provider_id: query.idp_id,
       idpartner_token: idpartnerToken,
-      claims: JSON.stringify(claimsObject),
       response_mode: 'jwt',
+      claims: JSON.stringify(claims),
+      ...otherAuthorizationParams,
```
Comment (yovasx2) [inline lib/idpartner.js:183]: you are gonna overwrite claims

**Review by yovasx2 — APPROVED** [review]

File: lib/idpartner.js (hunk 1 above)
Comment (francisco-idpartner) [reply lib/idpartner.js:183]: Yup, is this an action item ?

Comment (francisco-idpartner) [reply lib/idpartner.js:183]: I see, no, I extract the claims up in line 151. And, the data in `otherAuthorizationParams` are the remnants of that decronstruction

Comment (yovasx2) [reply lib/idpartner.js:183]: sweet!

**Pull request merged**